aptos-move/framework/aptos-framework/sources/account/account.move

```diff
@@ -1081,7 +1081,7 @@ module aptos_framework::account {
     #[test_only]
     public fun create_account_for_test(new_address: address): signer {
         // Make this easier by just allowing the account to be created again in a test
-        if (!exists_at(new_address)) {
+        if (!resource_exists_at(new_address)) {
```

Comment:
What changed here?

Reply:
if the feature is on, `exists_at` is always true.